### DISCUSSION TITLE
How to override platform controller

### DISCUSSION THREAD

**Discussion by Cregennan** [orchidsoftware/platform#2351] [Q&A]
I need to implement OAuth 2 in my admin panel. How to override `Orchid\Platform\Http\Controllers\LoginController` class so I can add custom auth logic?

**Comment by tabuna** [comment]
Provider classes are not meant to change. You can disable the login page in settings 
https://github.com/orchidsoftware/platform/blob/c0299a7d057c9030a0e953a006134751b2ca1b87/config/platform.php#L71-L86
 and instead make your own using packages or by yourself (For example, inherit)
